MO4/Sniffs/Formatting/UnnecessaryNamespaceUsageSniff.php

```diff
@@ -242,12 +242,17 @@ protected function getUseStatements(
                 break;
             }
 
+            // phpcs:disable
```

Comment:
Disabling all the phpcs rules is quire risky. I know that `phpcs` and other tools honors `@var` annotations in docblocks, but the default PHPCS codesniffer rules don't want  `/**` at all inlined doc blocks :disappointed: 

Either change this to 
```
            // phpcs:disable Squiz.Commenting.BlockComment.NoEmptyLineAfter

```
with a empty line after the `phpcs:disable`

or add a empty line after the comment block.

Or the `phpcs.xml.dist` need some updates on doc blocks being allowed inline.

Reply:
When disabling one sniff, a lot of otehr sniffs appear... also, type annotations are required in anotehr format. I kept it as is now. Very happy to move to MO4 also for the Sniff code, or at least away from Squiz.